mcm/tools/handlers.py

```diff
@@ -310,7 +310,10 @@ def make_injection_command(self, mcm_r=None):
         command += 'export X509_USER_PROXY=%s\n\n' % (proxy_file_name)
 
         executable_file_name = '%supload_script_%s.sh' % (directory, mcm_r.get_attribute('prepid'))
-        default_scram_arch = scram_arch.startswith('slc7_')
+
+        # Always use singularity containers.
+        default_scram_arch = False
```

Comment:
Reduce complexity by removing the variable altogether

Reply:
Fixed in the latest commit